app/package.json

```diff
@@ -28,9 +24,9 @@
     "fmt:fix": "prettier --write .",
     "format": "yarn nice",
     "ia": "yarn install-app",
-    "install-app": "yarn install-app:setup && cd ios && bundle exec pod install && cd .. && yarn clean:xcode-env-local",
+    "install-app": "yarn install-app:setup && cd ios && bundle install && bundle exec pod install && cd .. && yarn clean:xcode-env-local",
```

Comment:
we need to run `bundle install` AND `bundle exec pod install`

if not, we run into weird build pipeline errors:
https://github.com/selfxyz/self/actions/runs/16587650502/job/46916055932?pr=795